packages/api/src/api/helix/moderation/HelixAutoModSettings.ts

```diff
@@ -0,0 +1,99 @@
+import { DataObject, rawDataSymbol, rtfm } from '@twurple/common';
+
+/** @private */
+export interface HelixAutoModSettingsData {
+	broadcaster_id: string;
+	moderator_id: string;
+	overall_level: number | null;
+	disability: number;
+	aggression: number;
+	sexuality_sex_or_gender: number;
+	misogyny: number;
+	bullying: number;
+	swearing: number;
+	race_ethnicity_or_religion: number;
+	sex_based_terms: number;
+}
+
+/**
+ * The AutoMod settings of a channel.
+ */
+@rtfm<HelixAutoModSettings>('api', 'HelixAutoModSettings', 'broadcasterId')
+export class HelixAutoModSettings extends DataObject<HelixAutoModSettingsData> {
+	/**
+	 * The ID of the broadcaster for which the AutoMod settings are retrieved.
+	 */
+	get broadcasterId(): string {
+		return this[rawDataSymbol].broadcaster_id;
+	}
+
+	/**
+	 * The ID of a user that has permission to moderate the broadcaster's chat room.
+	 */
+	get moderatorId(): string {
+		return this[rawDataSymbol].moderator_id;
+	}
+
+	/**
+	 * The default AutoMod level for the broadcaster. This is null if the broadcaster changed individual settings.
+	 */
+	get overallLevel(): number | null {
+		return this[rawDataSymbol].overall_level ? this[rawDataSymbol].overall_level : null;
+	}
+
+	/**
+	 * The Automod level for discrimination against disability.
```

Comment:
I see that you have liberally mixed the capitalization of "AutoMod", please make it consistent :)

Reply:
Apologies, I should copy-paste less haha.